index.html

```diff
@@ -0,0 +1,62 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>einstein</title>
+</head>
+<main id="main">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Albert Einstein</h1>
+    <p>Pioneer Scientist in the field of physics</p>
+    <figure id="img-div">
+      <img
+        id="image"
+        src="einstein.jpg"
+        alt="einstein"
+      />
```

Comment:
Please put your images on a different folder :smiley: it makes your code more clean.

Reply:
Excellent